lib/companies/pending_changes.ex

```diff
@@ -138,9 +142,17 @@ defmodule Companies.PendingChanges do
     error
   end
 
+  defp apply_changes("create", changeset), do: Repo.insert(changeset)
+  defp apply_changes("update", changeset), do: Repo.update(changeset)
+  defp apply_changes("delete", record), do: Repo.delete(record)
+
   defp resource_module("company"), do: Company
   defp resource_module("industry"), do: Industry
   defp resource_module("job"), do: Job
+  defp resource_module(_), do: {:error, :module_not_recognized}
```

Comment:
This should never happen, we don't need to worry about it. What brought you here?